documentation/docs/99-legacy/00-legacy-overview.md

```diff
@@ -10,3 +10,5 @@ The following pages document these features for
 - people using Svelte 5, but with components that haven't yet been migrated
 
 Since Svelte 3/4 syntax still works in Svelte 5, we will distinguish between _legacy mode_ and _runes mode_. Once a component is in runes mode (which you can opt into by using runes, or by explicitly setting the `runes: true` compiler option), legacy mode features are no longer available.
+
+If you're exclusively interested in the Svelte 3/4 syntax, you can browse its documentation at [v4.svelte.dev](https://v4.svelte.dev).
```

Comment:
do you think it needs some kind of callout box?

Reply:
The page doesn't have much content, so I don't think so